prestodb/presto
# Annotate Live Plan with additional node info

Variant of #11927 which allows the output to be even more structured.

This change paves the way for PlanPrinter to generate more structured
output than a text blob. This structured output can then be used to
enrich a raw JSON plan.

Test plan:
- Run AbstractTestQueries with patch that dumps plan for every query, compare two dozen or so query plans with master.
- Run AbstractTestQueries with patch that does not catch exceptions thrown when constructing text plans, and verify that tests all pass.
